src/OrbitQt/CopyKeySequenceEnabledTreeView.h

```diff
@@ -6,6 +6,7 @@
 #define ORBIT_QT_COPYABLE_TREE_VIEW_H_
 
 #include <QKeyEvent>
+#include <QMouseEvent>
 #include <QTreeView>
 
 class CopyKeySequenceEnabledTreeView : public QTreeView {
```

Comment:
The name of this class was tightly coupled to the one signal it was adding. I knew the time would come when we would add another signal and make the name obsolete. So unless there is a way to do the `altKeyAndMousePressed` with only existing signals, we need to rename the class. 
I'm not super sure about the name though. `AdditionalSignalsTreeView`? `CustomSignalsTreeView`? 
We could also tie it to the fact that it's used for the "call tree"s and call it `CallTreeTreeView` but I think we can keep it generic, there is nothing in this class making it specific for the top-down and bottom-up views.